.github/workflows/build_binaries.yaml

```diff
@@ -0,0 +1,42 @@
+name: Build and publish binaries
+on:
+  release:
+    types:
+      - created
+jobs:
+  build:
+    name: Build and publish binaries
+    strategy:
+      matrix:
+        go-version: [ 1.16.x ]
```

Comment:
why not `1.17.x`?

Reply:
Because this PR is for building binaries using our existing Go version, the same one as we use in the Docker build.
If we would want to upgrade our Go version, this is possible but it should go in a different PR. But there is no need for it now.

Reply:
btw, Go 1.18 came out since you posted this comment :)